Algorithmia/__main__.py

```diff
@@ -0,0 +1,133 @@
+import sys
+import os
+import json
+import Algorithmia
+from pathlib import Path
+from Algorithmia.CLI import CLI
+
+import importlib
+importlib.reload(Algorithmia.CLI)
+
+
+#CLI app to allow a user to run algorithms and manage data collections
+
+usage = """CLI for interaction with Algorithmia
+Usage:
+algo [<cmd>] [options] [<args>...]
+algo[<cmd>] [--help | --version]
+
+General commands include:
+  auth	configure authentication
+
+  Algorithm commands include:
+    run	Runs an algorithm
+    clone	Clones an algorithm source
+
+  Data commands include:
+    ls
+    mkdir
+    rmdir
+    rm
+    cp
+    cat
+
+  Global options:
+    --help
+    --profile <name>
+"""
+
+def main():
+	args = sys.argv[1:]
+	#print(args)
+	
+	if len(args) < 1 or args[0] == "--help":
+		print(usage)
+	elif args[0] == "--version":
+		#print(VERSION)
+		print("version")
```

Comment:
I'm not sure if there is a good way to get the version. But if there is not, let's just remove this flag.